### PR TITLE
feat: add trust remote code to cross encoders

### PR DESCRIPTION
**Pull request by bwanglzu**
A tiny PR to allow `trust_remote_code` in `CrossEncoder` to align with `SentenceTransformer` signature.

Also a bit cleaning of the code, remove duplicate variables and re-order the docstring.

### CODE REVIEW

File: sentence_transformers/SentenceTransformer.py [hunk 1]
```diff
@@ -64,11 +64,11 @@ class SentenceTransformer(nn.Sequential):
     :param default_prompt_name: The name of the prompt that should be used by default. If not set,
         no prompt will be applied.
     :param cache_folder: Path to store models. Can also be set by the SENTENCE_TRANSFORMERS_HOME environment variable.
-    :param revision: The specific model version to use. It can be a branch name, a tag name, or a commit id,
-        for a stored model on Hugging Face.
     :param trust_remote_code: Whether or not to allow for custom models defined on the Hub in their own modeling files.
         This option should only be set to True for repositories you trust and in which you have read the code, as it
         will execute code present on the Hub on your local machine.
+    :param revision: The specific model version to use. It can be a branch name, a tag name, or a commit id,
```
Comment (tomaarsen) [inline sentence_transformers/SentenceTransformer.py:70]: Well spotted!

**Comment by bwanglzu** [comment]
@tomaarsen my colleague did some test let me try again :)

**Comment by bwanglzu** [comment]
you're right, i tested again, even through the results are identical, without adding `trust_remote_code` to `from_pretrained` keeps asking me to enter `yes` with a pop-up message in CLI:

```bash
tensor([0.2811, 0.3452, 0.8731])
The repository for jinaai/jina-reranker-v1-tiny-en contains custom code which must be executed to correctly load the model. You can inspect the repository content at https://hf.co/jinaai/jina-reranker-v1-tiny-en.
You can avoid this prompt in future by passing the argument `trust_remote_code=True`.

Do you wish to run the custom code? [y/N] y
[0.28105804 0.34524447 0.87314653]
```

after adding it:

```console
tensor([0.2811, 0.3452, 0.8731])
[0.28105804 0.34524447 0.87314653]
```

I updated the PR :) thanks!

Bo
